### DISCUSSION TITLE
I am getting this issue, i can not run the command mitype

### DISCUSSION THREAD

**Discussion by irffanasiff** [Mithil467/mitype#101] [Q&A] [answered]
➜  ~ `pip install mitype`
Collecting mitype
  Downloading mitype-0.2.3.tar.gz (664 kB)
     |████████████████████████████████| 664 kB 1.4 MB/s 
Building wheels for collected packages: mitype
  Building wheel for mitype (setup.py) ... done
  Created wheel for mitype: filename=mitype-0.2.3-py3-none-any.whl size=648834 sha256=354dd341112e54a67a5301f7fe4aceb9bbd42ed02711397beedc0dc44b0a640f
  Stored in directory: /home/irfan/.cache/pip/wheels/68/fd/24/fd8ac5331d55895f67d3b6a20fbe441339c5590441a3689149
Successfully built mitype
Installing collected packages: mitype
  *WARNING: The script mitype is installed in '/home/irfan/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.*
Successfully installed mitype-0.2.3
➜  ~ `mitype`       
zsh: command not found: mitype
➜  ~

**Comment by Mithil467** [comment, marked as answer]
As we can see in the warning, `/home/irfan/.local/bin` is not on PATH.
For zsh -
```
path+=('/home/<username>/.local/bin') # Add your username here
export PATH
```
Reference - [StackOverflow answer](https://stackoverflow.com/a/18077919/10273360).

Also, you can always run mitype without adding it to PATH with the help of this command -
`python -m mitype`
